packages/front-end/components/Layout/Layout.tsx

```diff
@@ -192,8 +193,37 @@ const backgroundShade = (color: string) => {
 
 const Layout = (): React.ReactElement => {
   const [open, setOpen] = useState(false);
+  const [isOldVersion, setIsOldVersion] = useState(false);
   const settings = useOrgSettings();
 
+  const build = getGrowthBookBuild();
+
+  function isCommitTooOld(remoteCommitDate: Date) {
+    if (!remoteCommitDate || !build.date) return true;
+    const msDiff = remoteCommitDate.valueOf() - new Date(build.date).valueOf();
+    const minuteDiff = msDiff / 60000;
+    const allowedMinuteDiff = 20;
```

Comment:
This is the wrong logic.  This is checking if the remote date is at least 20 minutes after the local build date.  What we want, is to just check that the latest build had enough time to finish

In the following scenario, we **don't** want to show the upgrade notice, even though the current build is 3 hours old since the new build is only 1 minute old and hasn't had enough time to finish yet.

- Current build: `8:00am`
- New build: `11:00am`
- Current date: `11:01am`

Meanwhile, we **do** want to show the notice here, even though the current build is only slightly older than the latest build.  The latest build has had 25 minutes to finish, which is all we care about.

- Current build: `11:00am`
- New build: `11:05am`
- Current date: `11:30am`